libs/market-list/src/lib/utils/market-list.utils.ts

```diff
@@ -0,0 +1,82 @@
+import BigNumber from 'bignumber.js';
+import type {
+  MarketList,
+  MarketList_markets,
+  MarketList_markets_candles,
+} from '../components/__generated__/MarketList';
+
+export const priceChangePercentage = (m: MarketList_markets): BigNumber => {
+  const change = priceChange(m);
+  const { candles } = m;
+  if (change && candles && candles.length > 0) {
+    const yesterdayLastPrice = price(candles[0]?.close, m.decimalPlaces);
+    return change.dividedBy(yesterdayLastPrice).multipliedBy(100);
+  }
+  return new BigNumber(0);
+};
+
+export const priceChange = ({
+  candles,
+  decimalPlaces,
+}: MarketList_markets): BigNumber => {
+  if (candles && candles.length > 0) {
+    const yesterdayLastPrice = price(candles[0]?.close, decimalPlaces);
```

Comment:
IMHO this function should remove price change only
`(candles[candles.length - 1]?.close !== undefined && candles[0]?.close !== undefined) ? 
BigInt(candles[candles.length - 1]?.close ?? 0)  - BigInt(candles[0]?.close ?? 0) : 0`
Then we can format it using formatNumber (and optinally PriceCell if we want to handle empty values)

Reply:
agree. updating